src/node/blockstorage.h

```diff
@@ -171,6 +174,7 @@ class BlockManager
     bool WriteUndoDataForBlock(const CBlockUndo& blockundo, BlockValidationState& state, CBlockIndex* pindex, const CChainParams& chainparams)
         EXCLUSIVE_LOCKS_REQUIRED(::cs_main);
 
+    /** Store block on disk. If dbp is not nullptr, then it provides the known position of the block within a block file on disk. */
```

Comment:
nit: Reading this comment it's not clear to me why this function is even called when the block is already on disk - it apparently doesn't need to be stored in that case.